ArchUnitNET/Domain/PlantUml/Export/PlantUmlSlice.cs

```diff
@@ -86,12 +105,44 @@ public string GetPlantUmlString(RenderOptions renderOptions)
                 }
             }
 
-            if (Hyperlink != null)
+            return result;
+        }
+
+        private StringBuilder BuildStringC4Style()
+        {
+            var result = new StringBuilder();
+            if (Namespace != null)
             {
-                result.Append(" [[" + Hyperlink + "]] ");
+                result.Append("Boundary(" + Namespace.Remove(Namespace.Length - 1) + ", " + Namespace.Remove(Namespace.Length - 1) + ") ");
```

Comment:
`Namespace.Remove(Namespace.Length - 1)` appears twice. Since `Remove` doesn't mutate the string, they are identical. If this is intended, I would suggest to extract the result, causes less confusion. Alternatively, you could also extract a (local) method and reuse it below.